packages/graphql-platform-connector-mariadb/src/graphql-platform/connector/database/operation/find.ts

```diff
@@ -207,7 +207,9 @@ export class FindOperation extends AbstractOperationResolver<
               break;
 
             case 'contains':
-              value !== null && where.addFilter(column, 'LIKE', `%${value}%`);
+              column.fullText !== null && value !== null && value.length >= column.config.fullText
+                ? where.addRaw(`MATCH(${column.name}) AGAINST('${value}' IN NATURAL LANGUAGE MODE)`)
+                : where.addFilter(column, 'LIKE', `%${value}%`);
```

Comment:
Almost :) But the `value !== null` must be outside the whole text, it does not have the same meaning here

```
value !== null && (
  column.fullText !== null && value.length >= column.config.fullText
    ? where.addRaw(`MATCH(${column.name}) AGAINST('${value}' IN NATURAL LANGUAGE MODE)`)
    : where.addFilter(column, 'LIKE', `%${value}%`)
);
```